src/util/simplify_expr.cpp

```diff
@@ -2382,6 +2382,7 @@ bool simplify_exprt::simplify_rec(exprt &expr)
   if(!simplify_node(tmp))
     result=false;
 
+#ifdef USE_LOCAL_REPLACE_MAP
   #if 1
```

Comment:
Therefore remove this `#if 1`?

Reply:
Well, there is still the alternative within this bit of code (it's part of an `#if 1 ... #else ... #endif`) so I don't think it should be removed?